src/config.ts

```diff
@@ -22,4 +23,10 @@ if (!ELASTICSEARCH_INDEX_NAME) {
     );
 }
 
-export { ELASTICSEARCH_URI, NATS_URL, ELASTICSEARCH_INDEX_NAME };
+if (!ELASTICSEARCH_INVENTORY_INDEX_NAME) {
```

Comment:
this won't be necessary anymore since we only need `ELASTICSEARCH_INDEX_NAME` to be set